deploy/kube-templates/heketi-deployment.yaml

```diff
@@ -49,6 +39,8 @@ spec:
         env:
         - name: HEKETI_EXECUTOR
           value: kubernetes
+        - name: HEKETI_KUBE_USE_SECRET
```

Comment:
I mean this is the default there are no other way for kubeapi authentication now from heketi.